classes/npm.bbclass

```diff
@@ -35,6 +35,8 @@ OWN_NPM_CLASS_PACKAGE ?= "0"
 # needed as gyp from bullseye does not establish /usr/bin/python
 NPM_EXTRA_DEPS = "${@'python' if d.getVar('NPM_REBUILD') == '1' else ''}"
 
+DEBIAN_BUILD_DEPENDS =. "${@'libnode-dev,' if d.getVar('NPM_REBUILD') == '1' else ''}"
+
```

Comment:
BTW, I cannot reproduce the problem: According to the issue description, `kas-container build kas-iot2050-example.yml --target node-red-node-serialport` should already trigger it today, without sbuilder, because there will be no mraa built in that case. But something else pulls it, indirectly. I wonder what that is.

Reply:
I rechecked it, monitoring buildchroot's `/var/cache/apt/archives` content at the same time. It seems that `libnode-dev` is installed by `do_install_npm()` (and not comes with `mraa` package that I first thought) - at least, `libnode-dev_12.22.5~dfsg-2~11u1_arm64.deb` file appeared here at this time. That's why buildchroot-based build works OK.

But in case of sbuild, build environment doesn't include libnode-dev, that's why an error appears.

Reply:
Here you can see the build log
[node-red-node-serialport_sbuild-log.txt](https://github.com/siemens/meta-iot2050/files/7948352/node-red-node-serialport_sbuild-log.txt)

Reply:
Actually, it seems that missing `libnode.so.72` library belongs to another package, `libnode72`, so the dependency tree could be smaller.

Reply:
Please clarify this before sending the split PR and also update the commit log of this patch accordingly.

Reply:
I rechecked it and four packages mentioned in previous pull-request (`node-red-contrib-modbus`, `node-red-node-serialport`, `node-red-node-sqlite` and `node-red`) are really depend on `libnode.so.72` library provided by `libnode72` package in bullseye.
So, we can add this dependency either to 4 recipes, or to npm.bbclass.

Reply:
Let's keep this injected globally via `NPM_REBUILD` condition. It only applies to nodes with native elements, but it's not worth to track that at node-level.

Reply:
OK